staff/lab/ocf-tv

```diff
@@ -43,15 +43,25 @@ def wait_for_port(host, port, timeout=5, ssh_proc=None):
 
 
 def volume(args):
-    call(('ssh', args.host,
-          'export PULSE_SERVER=localhost; pactl set-sink-volume {} {}%'.format(args.sink, args.amount)))
+    cmd = 'export PULSE_SERVER=localhost; '
+
+    if args.amount is not None:
+        cmd += 'pactl set-sink-volume 0 {}%; '.format(args.amount)
+
+    # awk prints the 9th line after the match
+    cmd += "pactl list sinks | awk '/Sink #0/{nr[NR+9]}; NR in nr' | cut -d' ' -f 6;"
```

Comment:
You can hold off for now and just document what it does as-is. It may break anyways when we upgrade to stretch.